plugins/modules/aws_kms.py

```diff
@@ -302,18 +305,23 @@
   returned: always
   sample: 1234567890123
 creation_date:
-  description: Date of creation of the key
+  description: Date and time of creation of the key
   type: str
   returned: always
   sample: "2017-04-18T15:12:08.551000+10:00"
+deletion_date:
```

Comment:
Don't we need to add version_added?

Reply:
This parameter was always returned, it just wasn't documented as being returned. Does it still need a version_added?

Reply:
updated!